src/sync/transforms/branch.ts

```diff
@@ -69,7 +70,7 @@ const mapCommit = (commit) => {
 		hash: commit.oid,
 		id: commit.oid,
 		issueKeys: issueKeys || [],
-		message: commit.message,
+		message: commit?.message?.substring(0, MAX_COMMIT_MESSAGE_LENGTH),
```

Comment:
`commit` will always be non-null.  the message value has to be non-null or else jira will reject it.  So maybe add a default in case?  `commit.message?.substring(0, MAX_COMMIT_MESSAGE_LENGTH) || ""`